packages/server/graphql/mutations/helpers/createTeamAndLeader.ts

```diff
@@ -69,4 +70,5 @@ export default async function createTeamAndLeader(userId: string, newTeam: Valid
   if (!organizationUser) {
     await adjustUserCount(userId, orgId, InvoiceItemType.ADD_USER)
   }
+  await insertOrgUserAudit([orgId], userId, 'added')
```

Comment:
why is this necessary? isn't this handled already in `adjustUserCount`? will calling it here create a dupe?

Reply:
it's not handled in `adjustUserCount` bc an `OrganizationUser` is inserted separately on line 44. Can we get rid of that insertion in the string of promises, and just call `adjustUserCount` to make it consistent?

Reply:
shoot i'm not seeing it. can you link to the link where `OrganizationUser` is getting inserted on L44?

Reply:
oops sorry, on line 44 it just fetches to check if an `OrganizationUser` was already made. `adjustUserCount` never gets called bc the check always finds an `OrganizationUser` already. 

it looks like `bootstrapNewUser` and `addOrg` both call `createOrg` which creates an `Organization` as well as `OrganizationUser` before calling `createTeamAndLeader`.

Reply:
maybe we can change `createNewOrg` to only create an `Organization` and not an `OrganizationUser`? it looks like `createNewOrg` is only called by `bootstrapNewUser` and `addOrg` and nothing else, so should be a safe change.

Reply:
ah, makes sense.
we don't want to do org user audit things in `createTeamAndLeader` because that can be called by `addTeam`, which would result in multiple records.
instead, let's
- remove L70-L73 here because that conditional branch is never true
- add the `insertOrgUserAudit` call to `createNewOrg`